src/utils/wallet.js

```diff
@@ -132,32 +132,50 @@ export class Wallet {
       return await this.account.removeAccessKey(this.accountId, publicKey)
    }
 
-   async checkAccount(accountId) {
-      if (accountId !== this.accountId) {
-         return await this.near.nearClient.viewAccount(accountId)
-      } else {
-         throw new Error('You are logged into account ' + accountId + ' .')
+   async checkAccountAvailable(accountId) {
+      if (!this.isLegitAccountId(accountId)) {
+         throw new Error()
```

Comment:
let's not ever throw errors without meaningful messages. It makes life harder for everyone trying to debug.

Reply:
fixed